extensions/eclipse-che-theia-remote-impl-che-server/tests/_data/devfile-kubernetes-components.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: hello
+projects:
+  - name: my-project
+    source:
+      type: git
+      location: 'https://github.com/this-is-a-test'
+      branch: myBranch
+components:
+  - type: kubernetes
+    alias: petclinic-db
+    reference: https://reference2
+    selector:
+      app.kubernetes.io/component: database
+  - type: kubernetes
```

Comment:
maybe it makes sense to have one component with type `openshift` in tests, it will cover this code block:
```typescript
} else if (componentV1.type === 'openshift') {
      devfileV2Component.openshift = {};
      devfileV2Component.openshift.inlined = JSON.stringify(componentV1);
}
```

Reply:
Will add it right now